pages/_app/hooks.js

```diff
@@ -3,7 +3,7 @@ import { join } from 'path'
 import { useEffect } from 'react'
 import { useRouter } from 'next/router'
 
-const useRedirect = (routePath, actualPath = routePath) => {
+const useRedirect = (routePath, actualPath = routePath, replace = true) => {
```

Comment:
Please, avoid boolean parameters without a name, it's not obvious. Imagine the call like `useRedirect('/hello', '/hello', false)`. What does it mean?

Prefer options instead: `useRedirect('/hello', '/hello', { replace: false })`, it looks better. However, repeated parameter value does not seem to be good still.

In this case, just do not add a parameter. We do not need it now and I am sure we will never need it.

Reply:
Doesn't your editor show you the parameter name?

Reply:
My one does not. I do no like that JetBrains' IDEs doing this because I used to know how long is my code (in columns) and it's annoying I cannot do it.

Also, GitHub does not do it, so for code review, it would be better to have a named boolean parameter. This is a good practice.

Reply:
Despite, I may overuse options. I wonder what you do think about.

Reply:
it makes sense to use them and you are right with Github. I am just lazy guy I would add it there once there is too many parameters.